test/Resolver/RevisionResolverTest.php

```diff
@@ -43,7 +43,7 @@ public function testGetRevisionAnnotation()
     public function testGetRevisionableFields()
     {
         $entity   = new \stdClass();
-        $metadata = $this->createMock('Doctrine\Common\Persistence\Mapping\ClassMetadata');
+        $metadata = $this->createMock('Doctrine\ORM\Mapping\ClassMetadata');
```

Comment:
Consider changing it to ClassMetadata::class. This way the error is much clearer for future cases. Also, it will be visible in the IDE when the class is tagged as deprecated.